esbuild.config.js

```
@@ -115,7 +116,25 @@ async function run() {
             '.woff2': 'file',
             '.eot': 'file',
         },
-        plugins: [rails(), sassPlugin(), vuePlugin, watchPlugin],
+        plugins: [
+            rails(),
+            sassPlugin({
+                importer(url, prev) {
+                  if (url.startsWith("~")) {
```

ESLint

[error] Expected indentation of 20 spaces but found 18.

(indent)